feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -222,6 +223,10 @@ class BillingWrapper(
         activity: Activity,
         params: BillingFlowParams
     ) {
+        if (activity.intent == null) {
+            activity.intent = Intent()
```

Comment:
Looks like devs might be setting the intent to null intentionally https://github.com/RevenueCat/purchases-android/issues/381#issuecomment-977393498, so I wonder if this will have unintended consequences for this developer.

Maybe we can ask them?

Reply:
asked!

Reply:
with "always be shipping" in mind, i've reverted the "fix" and just left the log. we can consider setting to a null intent once we hear back.

seems like a react-native plugin also sets the intent to null, and someone posted a fix: https://github.com/ajith-ab/react-native-receive-sharing-intent/issues/110#issuecomment-971612714

but i still don't feel confident whether that could lead to unwanted side effects. i thought about if we could set it to Intent() before calling `launchBillingFlow`, and then set it back to null after (if it was null)? still probably not foolproof though...